sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sfas-files/SFAS-TO-SIMS-2024JUL09-INVALID-INDIVIDUAL-RECORDS.txt

```diff
@@ -0,0 +1,5 @@
+100AESTSFAS TO SIMS BRIDGE                    20240709110630                                                                                                                                                                                              

```

Comment:
Minor. I do not see a significance of using the data to the file name. I know it is influenced from existing file names. But IMO this could be as simple as `SFAS-TO-SIMS-INVALID-INDIVIDUAL-RECORDS.txt`

Same applies to other file as well.